frontend/src/app/testResults/resultsTable/ResultsTable.tsx

```diff
@@ -169,94 +130,97 @@ const generateResultRows = (
-        setPrintModalId,
-        r,
-        setEmailModalTestResultId,
-        setTextModalId,
-        removed,
-        setMarkCorrectionId,
-        setDetailsModalId
-      );
-      const getResultCellHTML = () => {
-        return (
-          <td
-            key={`${r.internalId}-${diseaseIdName}`}
-            className="test-result-cell"
-            data-testid={`${diseaseIdName}-result`}
-          >
-            {r.testResult ? TEST_RESULT_DESCRIPTIONS[r.testResult] : "N/A"}
-          </td>
-        );
-      };
+  return sortedResults.map((r: Result) => {
+    const removed = r.correctionStatus === "REMOVED";
+
+    let diseaseIdName = toLowerCaseHyphenate(r.disease);
+
+    const actionItems = createActionItemList(
+      setPrintModalId,
+      r,
+      setEmailModalTestResultId,
+      setTextModalId,
+      removed,
+      setMarkCorrectionId,
+      setDetailsModalId
+    );
+    const getResultCellHTML = () => {
       return (
-        <tr
-          key={`${r.internalId}-${diseaseIdName}`}
-          title={removed ? "Marked as error" : ""}
-          className={classnames(
-            "sr-test-result-row",
-            removed && "sr-test-result-row--removed"
-          )}
-          data-testid={`test-result-${r.internalId}-${diseaseIdName}`}
-          data-patient-link={
-            r.patientLink
-              ? `${getUrl()}pxp?plid=${r.patientLink.internalId}`
-              : null
-          }
+        <td
+          key={`${r.id}-${diseaseIdName}`}
+          className="test-result-cell"
+          data-testid={`${diseaseIdName}-result`}
         >
-          <td className="patient-name-cell">
-            <Button
-              variant="unstyled"
-              ariaLabel={formatTestResultAriaLabel(r)}
-              label={displayFullName(
-                r.patient?.firstName,
-                r.patient?.middleName,
-                r.patient?.lastName
-              )}
-              onClick={() => setDetailsModalId(r.internalId)}
-              className="sr-link__primary"
-            />
-            <span className="display-block text-base font-ui-2xs">
-              DOB: {formatDateWithTimeOption(r.patient?.birthDate)}
-            </span>
-          </td>
-          <td className="test-date-cell">
-            {formatDateWithTimeOption(r.dateTested, true)}
+          {r.testResult ? TEST_RESULT_DESCRIPTIONS[r.testResult] : "N/A"}
+        </td>
+      );
+    };
+    return (
+      <tr
+        key={`${r.id}-${diseaseIdName}`}
+        title={removed ? "Marked as error" : ""}
+        className={classnames(
+          "sr-test-result-row",
+          removed && "sr-test-result-row--removed"
+        )}
+        data-testid={`test-result-${r.id}-${diseaseIdName}`}
+        data-patient-link={null}
+        /*
+        data-patient-link={
+          r.testOrder.patientLink
+            ? `${getUrl()}pxp?plid=${r.testOrder.patientLink.internalId}`
+            : null
+        }
+         */
```

Comment:
remove this commented out code

Reply:
👍 Removed in https://github.com/CDCgov/prime-simplereport/pull/6682/commits/34346c5a58c35cca7a4fde47695fcf9e994ce8ae.